package.json

```diff
@@ -35,11 +35,11 @@
     "local-publish": "node ./scripts/verdaccio-publish/index.js",
     "test:all": "yarn build:all && jest --coverage --passWithNoTests && lerna run test --scope '@aws-sdk/{fetch-http-handler,hash-blob-browser}' && yarn test:versions && yarn test:integration",
     "test:ci": "lerna run test --since origin/main",
-    "test:e2e": "yarn build:e2e && node ./tests/e2e/index.js",
+    "test:e2e": "node ./tests/e2e/index.js",
+    "test:e2e:legacy": "cucumber-js --fail-fast",
+    "test:e2e:legacy:since:release": "./tests/integ-e2e/index.js",
     "test:functional": "jest --passWithNoTests --config tests/functional/jest.config.js && lerna run test:unit --scope \"@aws-sdk/client-*\"",
     "test:integration": "jest --config jest.config.integ.js --passWithNoTests",
-    "test:integration:legacy": "cucumber-js --fail-fast",
-    "test:integration:legacy:since:release": "./tests/integ-legacy/index.js",
```

Comment:
Retain this duplicate scripts as of now.
The internal builds will start failing between the time this commands are removed, and internal release changes reach production.